addons/sales_team/static/src/js/crm_team_form.js

```diff
@@ -0,0 +1,47 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { formView } from "@web/views/form/form_view";
+import { FormController } from "@web/views/form/form_controller";
+import { user } from "@web/core/user";
+import { useService } from "@web/core/utils/hooks";
+import { registry } from "@web/core/registry";
+
+
+class CrmTeamFormController extends FormController {
+
+    setup() {
+        super.setup();
+        this.orm = useService("orm");
+    }
+
+    async beforeExecuteActionButton(clickParams) {
+        if (clickParams.name === "activate_multi_membership") {
+            if (!user.hasGroup("sales_team.group_sale_manager")) {
+                return false;
+            }
+            const alert = document.querySelector(".o_crm_team_member_alert");
```

Comment:
Can't we find the alert and not use a custom class ? Normally you won't have other alerts, do we ?